inc/classes/class-assets.php

```diff
@@ -45,9 +56,9 @@ protected function _setup_hooks() {
 	 */
 	public function register_scripts() {
 
-		wp_register_script( 'blank-theme-main', BLANK_THEME_BUILD_URI . '/js/main.js', [ 'jquery' ], false, true );
-		wp_register_script( 'blank-theme-home', BLANK_THEME_BUILD_URI . '/js/home.js', [ 'jquery' ], false, true );
-		wp_register_script( 'blank-theme-single', BLANK_THEME_BUILD_URI . '/js/single.js', [ 'jquery' ], false, true );
+		wp_register_script( 'blank-theme-main', $this->get_asset_file_uri( 'main.js' ), [ 'jquery' ], filemtime( $this->get_asset_file_path( 'main.js' ) ), true );
+		wp_register_script( 'blank-theme-home', $this->get_asset_file_uri( 'home.js' ), [ 'jquery' ], filemtime( $this->get_asset_file_path( 'home.js' ) ), true );
+		wp_register_script( 'blank-theme-single', $this->get_asset_file_uri( 'single.js' ), [ 'jquery' ], filemtime( $this->get_asset_file_path( 'single.js' ) ), true );
```

Comment:
`filemtime()` don't needed now when we use hashed file.

Reply:
for develop build we are not using hashed files, hence if we decide to run develop build on any of the servers, as we do pre launch, we may need this filemtime(), 

Other way is that I can update to use hashes for dev builds as well, then we won't need to rely on filemtime(), 

Just trying to understand if there was any particular reason behind keeping dev build file name without hashes, @manishsongirkar

Reply:
keeping the `filemtime()` for now, WPCS may flag if we keep the version value default or to false.